Front-End/src/components/Register.js

```diff
@@ -0,0 +1,84 @@
+import React,{useState} from 'react'
+import logo from './pics/logo.png'
+import axios from 'axios'
+import {useHistory} from 'react-router-dom';
+
+const Register = () => {
+
+const history = useHistory();    
+const [userName,setuserName]=useState('')
+const [email,setemail]=useState('')
+const [password,setpassword]=useState('')
+const [confPassword,setconfPassword]=useState('')
+const [phone,setphone]=useState('')
+const [adress,setadress]=useState('')
+const [role,setrole]=useState('student')
+const [emailErr,setemailErr]=useState('')
+const [userErr,setuserErr]=useState('')
+const [passErr,setpassErr]=useState('')
+
+const roleHandler=()=>{
+    setrole("instructor")
+}
+const passMatch=(password,confPassword)=>{
+    if(password !== confPassword){
+        setpassErr(true);
+        console.log(passErr)
+        console.log("not match")
+    }
+}
+const onSignUp=(name,email,adress,password,phone)=>{
+    console.log(passErr)
+    if(passErr) return "error"
```

Comment:
Remove the console log, prevent the user from signing up by disabling the button if the passwords do not match